### DISCUSSION TITLE
Scene Meter

### DISCUSSION THREAD

**Discussion by keeper2597** [leolabs/ableton-js#72] [Ideas]
Hi - would it be possible to make meter a gettable and settable property (like tempo) for Scenes?

Thanks again for the great work!

**Comment by keeper2597** [comment]
Actually, I see that maybe meter is not part of the Live Object Model for Scenes.  I know in Ableton 9 this was in the scene name.  Wonder where it's stored now??

**Comment by keeper2597** [comment]
I see that in the .als file a scene encodes the meter information like this:
```
<TimeSignatureId Value="201" />
<IsTimeSignatureEnabled Value="false" />
```

**Comment by leolabs** [comment]
I just checked the API for Scene objects and it seems that unfortunately, Live doesn't expose the `meter` information at all. You could try to read it from the project file directly, but not sure how viable of a solution that would be.
